app/templates/views/service-settings/service-already-live.html

```diff
@@ -0,0 +1,24 @@
+{% extends "withnav_template.html" %}
+{% from "components/page-header.html" import page_header %}
+
+{% block service_page_title %}
+  Your service is already live
+{% endblock %}
+
+{% block maincolumn_content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      {{ page_header('Your service is already live') }}
+
+      <p class="govuk-body">
+        {{ current_service.name }} is already live.
+      </p>
```

Comment:
The h1 and first line of body content are very similar. We should avoid repetition if we can.

**Solution 1** – make the service name part of the h1

```suggestion
      {{ page_header('‘{{ current_service.name }}’ is already live') }}
```

Issue: we know that some service names can be very long

**Option 2** – remove the service name altogether

```suggestion
      {{ page_header('Your service is already live') }}
```

Issue: while this looks neater, it’s less helpful – especially if the user doesn’t notice the current service name in the breadcrumb.

Reply:
We discussed another option, which was to make the first line of the body more useful. For example:

```‘Service name’ went live on Thursday 5 November 2020.```

We didn‘t go with this because it would make the code more complicated for a page which is very rarely seen (@klssmith can only remember seeing this happen twice).

Reply:
Actually, I've just taken another look and it is a very easy change to add the date and doesn't complicate things after all.

Reply:
Great, let’s go with:

```suggestion
      {{ page_header('Your service is already live') }}

      <p class="govuk-body">
        ‘{{ current_service.name }}’ went live on [day] [date] [month] [year].
      </p>
```

Reply:
Ok, that's updated now - I've updated the screenshot to show the new message.